contents/docs/surveys/manual.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Surveys (beta)
+---
+
+Surveys are a great way to collect qualitative feedback from your users. Qualitative feedback allows you to gain more descriptive insight into your users' experience.
+
+With surveys you'll be able to ask your users questions, target based on page url, selectors, feature flag, and user properties, and then analyze the results.
+
+## Getting started
+
+Setting up surveys is easy! We recommend option 1 if you're just getting started, but if you want to build your own custom survey components, option 2 will let you do that.
+
+### Option 1 (recommended)
+
+1. Make sure you've installed and enabled the surveys app in your project's apps.
```

Comment:
I'm not fully au fait with how apps are changing at the moment, but we may need to change the language here to make this distinct from apps/integrations/plugins/cdp. 

We should also make these clickable - e.g:

Reply:
```suggestion
1. Make sure you've installed and enabled the surveys app in [your project's apps](https://app.posthog.com/apps).
```

Reply:
This won't be a CDP pipeline, it will definitely be an app.